lxc/config_trust.go

```diff
@@ -159,9 +174,73 @@ func (c *cmdConfigTrustAdd) Run(cmd *cobra.Command, args []string) error {
 		cert.Projects = strings.Split(c.flagProjects, ",")
 	}
 
+	if cert.Token {
+		op, err := resource.server.CreateCertificateToken(cert)
+		if err != nil {
+			return err
+		}
+
+		if !c.global.flagQuiet {
+			opAPI := op.Get()
+			joinToken, err := certificateAddTokenOperationToAPI(&opAPI)
+			if err != nil {
+				return fmt.Errorf("Failed converting token operation to join token: %w", err)
+			}
+
+			fmt.Printf(i18n.G("Client %s join token: %s")+"\n", cert.Name, joinToken.String())
+		}
+
+		return nil
+	}
+
 	return resource.server.CreateCertificate(cert)
 }
 
+func certificateAddTokenOperationToAPI(op *api.Operation) (*api.CertificateAddToken, error) {
```

Comment:
Maybe make this a function on an `api.Operation`?

Like `op.ToCertificateAddToken(*CertificateAddToken, error)`?

Reply:
I don't mind. Should we do this for the cluster join tokens as well then? Currently that uses `clusterJoinTokenOperationToAPI(op *api.Operation) (*api.ClusterMemberJoinToken, error)`.

Reply:
Yep, that'd be great

Reply:
Done